File: PolyclinicsSystemBackend/PolyclinicsSystemBackend/Data/Entities/MedicalCard/MedicalCard.cs
using System.Collections.Generic;

namespace PolyclinicsSystemBackend.Data.Entities.MedicalCard
{
    public class MedicalCard
    {
        public int MedicalCardId { get; set; }

        public string UserId { get; set; }

        public User User { get; set; }

        public string? AdditionalInfo { get; set; }

        public Genders? Gender { get; set; }

        public int? Height { get; set; }

        public double? Weight { get; set; }

        public int? Age { get; set; }

        public List<Diagnose> Diagnoses { get; set; }
    }
}

File: PolyclinicsFrontEnd/src/app/shared/models/medical-card/medical-card.ts
import {Diagnose} from "../diagnose/diagnose";
import {Gender} from "../../enums/gender";

export class MedicalCard {
  public medicalCardId: number = -1
  public gender: Gender = Gender.Male
  public additionalInfo: string = ''
  public height: number = -1
  public weight: number = -1
  public age: number = -1
  public diagnoses: Array<Diagnose> = new Array<Diagnose>()
}
